README.md

```diff
@@ -18,7 +16,7 @@ If you are passionate about iOS and kernel exploitation and want to help us make
 
 ---
 To start the process we first need to prepare a kernel image, a device tree, a static trust cache, the main and the secondary disk images.
-To get the all that we first need to get the update file from Apple: [iOS 12.1 update file].
+To get the all that, we first need to get the update file from Apple: [iOS 12.1 update file].
```

Comment:
Thanks for noticing. Let's also remove the `the`, which shouldn't be here in the first place (To get ~~all~~ that)